ChangeLog.md

```diff
@@ -128,6 +128,8 @@
   * Remove `element_index` from list of arguments passed to elements (Tim Bretl).
 
   * Change `pl-string-input` to include an attribute for the placeholder (Mariana Silva).
+  
+   * Change element documentation to add placeholder attribute to pl-string-input (Mariana Silva).
```

Comment:
Something looks wrong with the spacing here? Also, if you put `pl-string-input` in backtick-quotes then it will look better in markdown.

Reply:
I used a different text editor and didn't notice the spacing issue. I will fix this and push again